integration/kubernetes/k8s-uts+ipc-ns.bats

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
```

Comment:
Nit: I'd remove this as strictly it isn't a bash script any more.

Reply:
maybe change it to: #!/usr/bin/env bats